installer.sh

```diff
@@ -89,13 +90,15 @@ cd $DIR
 wget https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.2.tar.gz
 tar xzf SDL2_mixer-2.0.2.tar.gz
 cd SDL2_mixer-2.0.2
-
-#SMPEG
-cd external/smpeg2-2.0.0
 ./configure
 make -j4
 make install -j4
-cd ../../
+cd $DIR
+
+#SMPEG
+wget https://www.libsdl.org/projects/smpeg/release/smpeg2-2.0.0.tar.gz
+tar xzf smpeg2-2.0.0.tar.gz
+cd smpeg2-2.0.0
```

Comment:
why did this change?

Reply:
For me smpeg didn't exist in the previous section

Reply:
Actually, same with me. My installer always stopped at 
```
./installer.sh: line 94: cd: external/smpeg2-2.0.0: No such file or directory
```

My usual fix was to just ignore it and comment it out, and re-run the script.